website/integrations/services/weblate/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Weblate
+---
+
+## What is Weblate
+
+From https://weblate.org/en/
+
+:::note
+Weblate is a copylefted libre software web-based continuous localization system, used by over 2500 libre projects and companies in more than 165 countries.
+:::
+
+
+## Preparation
+
+The following placeholders will be used:
+
+- `weblate.company` is the FQDN of the Weblate install.
+- `authentik.company` is the FQDN of the authentik install.
+- `weblate-slug` is the slug of the Weblate application
+
+Create an application in authentik and note the slug, as this will be used later. Create a SAML provider with the following parameters:
+
+- ACS URL: `https://weblate.company/accounts/complete/saml/`
+- Audience: `https://weblate.company/accounts/metadata/saml/`
+- Service Provider Binding: Post
+- Issuer: `https://authentik.company/application/saml/weblate-slug/sso/binding/redirect/`
+
+You can of course use a custom signing certificate, and adjust durations.
+
+## Property mappings
+
+We need to create some property mappings so our application will work. After you create the property mappings, assign them to the provider.
+
+### Full name
+* Name: `Weblate - Full name`
+* SAML Attribute Name: `urn:oid:2.5.4.3`
```

Comment:
I assume these urn attributes are hardcoded in weblate? Seeing as most of these custom mappings duplicate inbuilt mappings

Reply:
Not in Weblate, but in social-core plugin. Without the manually created properties, the process does not work. 

*  https://github.com/python-social-auth/social-core/blob/32d7bcf41dd0f21dc71cf14cb5a7373a7e522e88/social_core/backends/saml.py#L45

These provide extra information, so we can provide more details about the user connected. Without this, the new user created has no extra information other than the email.

* https://github.com/python-social-auth/social-core/blob/32d7bcf41dd0f21dc71cf14cb5a7373a7e522e88/social_core/backends/saml.py#L17-L23
* https://github.com/python-social-auth/social-core/blob/32d7bcf41dd0f21dc71cf14cb5a7373a7e522e88/social_core/backends/saml.py#L51-L67